mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/VirksomhetService.kt

```diff
@@ -36,13 +37,15 @@ class VirksomhetService(
 
     suspend fun syncEnhetFraBrreg(orgnr: String): VirksomhetDto {
         val enhet = CacheUtils.tryCacheFirstNotNull(brregServiceCache, orgnr) {
-            brregClient.hentEnhet(orgnr)
+            brregClient.hentEnhet(orgnr).getOrElse { throw it }
```

Comment:
Hvis exception uansett kastes her så kan vi kanskje gjøre det rett fra klienten i stedet for å returnere `Either<Exception, VirksomhetDto>`?

Reply:
Ja, det var opprinnelig kastet rett fra klienten. Planen var å gjøre feilhåndtering dersom vi ikke får kontakt med brreg på en bedre måte, men jeg er litt usikker på hva jeg egentlig vil skal skje...